components/usage/pkg/scheduler/reporter.go

```diff
@@ -16,26 +16,26 @@ const (
 )
 
 var (
-	reconcileStartedTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+	jobStartedSeconds = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
These are now general job metrics, rather than specific for the reconcile with ledger. If a job desires more metrics, those can be added in job direclty rather than on the scheduler level.